lms/djangoapps/verify_student/urls.py

```diff
@@ -152,3 +152,11 @@
         name="verify_student_incourse_reverify"
     ),
 )
+
+if settings.FEATURES.get('ENABLE_SOFTWARE_SECURE_FAKE'):
+    # Fake response page for incourse reverification ( software secure )
+    from verify_student.tests.fake_software_secure import SoftWareSecureFakeView
```

Comment:
https://github.com/edx/edx-platform/blob/master/lms/djangoapps/shoppingcart/urls.py#L24
I think view is enabling only if the feature is true.